elastic/apm-server
# Fix rum rate limit default value

## Motivation/summary

Map the cloud default value for `apm-server.rum.rate_limit` to the newer config setting `apm-server.rum.event_rate.limit` for the switch to data streams. 



## How to test these changes
Initiate switch to data streams in the Kibana UI on a cloud deployment and ensure the `apm-server.rum.rate_limit` setting does not show up as unsupported.
